index.html

```diff
@@ -19,12 +20,62 @@
 
 <body>
 
-	<i id="switch-task-button" class="fa fa-bars navbar-action-button theme-text-color windowDragHandle" title="View Tasks"></i>
+	<i id="switch-task-exit-button" class="fa fa-bars navbar-action-button active theme-text-color windowDragHandle" title="View Tasks" hidden></i>
+
+	<div class="titlebar-windows">
+		<div class="element caption-minimise">
```

Comment:
Did you make the icons yourself, or did you get them from somewhere else? If so, it might be good to add a comment with a link to the original source.